packages/experiments-realm/Product/4c2a356e-ec74-4a49-9aa2-a88df64cf446.json

```diff
@@ -31,12 +31,12 @@
       },
       "unitPrice.currency": {
         "links": {
-          "self": "../Currency/2"
+          "self": "http://localhost:4201/experiments/Currency/2"
```

Comment:
don't use absolute URL's. probably this is the result of a bug, do you recall how this was altered?

Reply:
No, but I suspect this: https://github.com/cardstack/boxel/blob/969a188918ffbf5aeaf083c5932cafee196d47cc/packages/experiments-realm/monetary-amount.gts#L17